packages/nimble-components/src/table-column/number-text/tests/table-column-number-text.stories.ts

```diff
@@ -185,6 +216,11 @@ export const numberTextColumn: StoryObj<NumberTextColumnTableArgs> = {
             options: [undefined, 0, 1, 2, 3, 20],
             control: { type: 'select' }
         },
+        unit: {
+            description: unitDescription,
+            options: ['default', 'byte', 'byte (1024)', 'volt'],
```

Comment:
Went with volt ⚡ (partially because the current setup makes for not great byte values, i.e. bytes usually aren't continuous and -6.45326E-7 bytes which the table will show right now doesn't make sense, but if interested we can clean-up in a follow-up)